scripts/nomad-dev.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+require() {
+  if ! hash "$1" &>/dev/null; then
+    echo "'$1' not found in PATH"
+    exit 1
+  fi
+}
+
+require nomad
+require consul
+require bindle-server
+require traefik
+
+trap 'kill $(jobs -p)' EXIT
+
+echo "Starting consul..."
+consul agent -dev -bootstrap-expect 1 -client '0.0.0.0' &>/tmp/consul.log &
+
+echo "Starting nomad..."
+nomad agent -dev &>/tmp/nomad.log &
+
+echo "Starting traefik..."
+traefik --entryPoints.http.address=:8088 --providers.consulCatalog &>/tmp/traefik.log &
+
+echo "Starting bindle..."
+bindle-server --unauthenticated &>/tmp/bindle.log &
+
+echo "Waiting for nomad..."
+while ! nomad server members 2>/dev/null | grep -q alive; do
+  sleep 2
+done
+
+dotnet run --project src/Web \
+  --Scheduler:Driver=nomad \
+  --Bindle:Url=http://localhost:8080/v1 \
+  --Nomad:Driver=raw_exec \
+  --Hippo:PlatformDomain='local.fermyon.link'
```

Comment:
I think this may be an issue because this is a deislabs project, not a fermyon project.

Perhaps `127.0.0.1.sslip.io` would be a better fit here?

Reply:
I'll try out using `sslip.io`